shells/dev/app/index.js

```diff
@@ -32,6 +33,7 @@ function mountTestApp() {
   mountHelper(ElementTypes);
   mountHelper(EditableProps);
   mountHelper(DeeplyNestedComponents);
+  mountHelper(SuspenseTree);
```

Comment:
nit Move this above `DeeplyNestedComponents` so we can test it without as much scrolling 😆

Reply:
I figured maybe you test some other case more often. But fair re: DeeplyNested ones. :-)